app/src/main/res/values/preference_keys.xml

```diff
@@ -80,7 +80,7 @@
     <string name="pref_key_screen_exceptions" translatable="false">
         <xliff:g id="preference_key">pref_screen_exceptions</xliff:g>
     </string>
-    <string name="pref_cfr_visibility_for_shield_toolbar_icon" translatable="false"><xliff:g id="preference_key">pref_cfr_visibility_for_shield_toolbar_icon</xliff:g></string>
+    <string name="pref_cfr_visibility_for_tracking_protection" translatable="false"><xliff:g id="preference_key">pref_cfr_visibility_for_tracking_protection</xliff:g></string>
```

Comment:
Same as above. We'd want to avoid renaming sharedPreference keys to not loose data.